assets/css/combined.css

```
@@ -1,4 +1,930 @@
+  .followerBanner ul li:first-child {
+    border-right: 1px solid #fff;
+  }
+}
+.followerBanner ul li a {
+  display: block;
+  padding: 20px 20px;
+  background-color: #2454C0;
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)